src/components/controller/units/UnitsTable.vue

```diff
@@ -417,6 +420,13 @@ function showRemoveUnitModal(unit: Unit) {
               </div>
             </div>
           </NeTableCell>
+          <NeTableCell>
+            <span class="truncate">
+              {{
+                item.groups.length > 0 ? item.groups.map((group: string) => group).join(', ') : '-'
+              }}
```

Comment:
probable trying to read `.length` of undefined

```suggestion
              <template v-if="item.groups == undefined || item.groups.length < 1"> - </template>
              <template v-else>
                {{ item.groups.map((group: string) => group).join(', ') }}
              </template>
```

Reply:
Already fixed